plugins/topology/src/components/Topology/TopologySideBar/TopologyCronJobDetails.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { V1CronJob } from '@kubernetes/client-node';
+import TopologySideBarDetailsItem from './TopologySideBarDetailsItem';
+import TopologyWorkloadDetails from './TopologyWorkloadDetails';
+import { Timestamp, TimestampFormat } from '@patternfly/react-core';
+
+const TopologyCronJobDetails: React.FC<{ resource: V1CronJob }> = ({
+  resource,
+}) => {
+  return (
+    <TopologyWorkloadDetails resource={resource}>
+      <TopologySideBarDetailsItem label="Schedule">
+        {resource.spec?.schedule}
+      </TopologySideBarDetailsItem>
+      <TopologySideBarDetailsItem label="Concurrency policy">
+        {resource.spec?.concurrencyPolicy}
+      </TopologySideBarDetailsItem>
+      <TopologySideBarDetailsItem label="Starting deadline seconds">
+        {resource.spec?.startingDeadlineSeconds
+          ? `${resource.spec.startingDeadlineSeconds} second`
```

Comment:
should we handle plural as well i.e second or seconds?

Reply:
It is just `second` in ODC but handled it here.